scripts/config-uart-console.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Remove any console definitions on serial0
+sed -i 's/ console=serial0,115200//' /boot/cmdline.txt
```

Comment:
isn't it better to use the regex here too?

Reply:
Yes